components/accordion/index.css

```diff
@@ -120,19 +178,152 @@ governing permissions and limitations under the License.
+  --spectrum-accordion-item-header-bottom-to-text-space: var(--spectrum-accordion-bottom-to-text-spacious-small);
+}
+
+.spectrum-Accordion-spacious--sizeM {
+  --spectrum-accordion-item-header-top-to-text-space: var(--spectrum-accordion-top-to-text-spacious-medium);
+  --spectrum-accordion-item-header-bottom-to-text-space: var(--spectrum-accordion-bottom-to-text-spacious-medium);
+}
+
+.spectrum-Accordion-spacious--sizeL {
+  --spectrum-accordion-item-header-top-to-text-space: var(--spectrum-accordion-top-to-text-spacious-large);
+  --spectrum-accordion-item-header-bottom-to-text-space: var(--spectrum-accordion-bottom-to-text-spacious-large);
+}
+
+.spectrum-Accordion-spacious--sizeXL {
+  --spectrum-accordion-item-header-top-to-text-space: var(--spectrum-accordion-top-to-text-spacious-extra-large);
+  --spectrum-accordion-item-header-bottom-to-text-space: var(--spectrum-accordion-bottom-to-text-spacious-extra-large);
+}
+
 .spectrum-Accordion-itemContent {
-  padding-block: 0 var(--spectrum-accordion-item-content-padding);
-  padding-inline: var(--spectrum-accordion-item-content-padding)
-    var(--spectrum-accordion-item-content-padding);
+  padding-block:  var(--mod-accordion-item-content-area-top-to-content, var(--spectrum-accordion-item-content-area-top-to-content)) var(--mod-accordion-item-content-area-bottom-to-content, var(--spectrum-accordion-item-content-area-bottom-to-content));
+  padding-inline: var(--mod-accordion-component-edge-to-text, var(--spectrum-accordion-component-edge-to-text)) var(--mod-accordion-component-edge-to-text, var(--spectrum-accordion-component-edge-to-text));
   display: none;
+  color: var(--mod-accordion-item-content-color, var(--spectrum-accordion-item-content-color));
+  font-weight: var(--mod-accordion-item-content-font-weight, var(--spectrum-accordion-item-content-font-weight));
+  font-style: var(--mod-accordion-item-content-font-style, var(--spectrum-accordion-item-content-font-style));
+  font-size: var(--mod-accordion-item-content-font-size, var(--spectrum-accordion-item-content-font-size));
+  font-family: var(--mod-accordion-item-content-font, var(--spectrum-accordion-item-content-font));
+  line-height: var(--mod-accordion-item-content-line-height, var(--spectrum-accordion-item-content-line-height));
+}
+
+.spectrum-Accordion-item {
+  border-color: var(--mod-accordion-divider-color, var(--spectrum-accordion-divider-color));
+  border-width: var(--mod-accordion-divider-thickness, var(--spectrum-divider-thickness-small));
+}
+
+.spectrum-Accordion-itemIndicator {
+  color: var(--mod-accordion-item-header-color-default, var(--spectrum-accordion-item-header-color-default));
+}
+
+.spectrum-Accordion-itemHeader {
+  color: var(--mod-accordion-item-header-color-default, var(--spectrum-accordion-item-header-color-default));
+  border-radius: var(--mod-accordion-corner-radius, var(--spectrum-accordion-corner-radius));
```

Comment:
Just noting that this `border-radius` is having an effect on the `:hover` state that makes it look rounded. I'm not sure that this is the desired effect. I looked at the spec file, and I think the only thing that might need a corner radius is the focus indicator.

<img width="148" alt="Screenshot 2023-05-12 at 4 08 29 PM" src="https://github.com/adobe/spectrum-css/assets/360251/e92db501-7674-40dc-af2a-df5624a4fe8a">

Reply:
Additionally, the spec file shows the disclosure indicator icon as inset from the edge a bit. I think you'd want to use `--spectrum-accordion-edge-to-disclosure-indicator` for this.

<img width="801" alt="Screenshot 2023-05-12 at 4 24 31 PM" src="https://github.com/adobe/spectrum-css/assets/360251/405ee530-9f17-4943-a7a2-fe08b53a1410">

Reply:
--spectrum-accordion-edge-to-disclosure-indicator is of value 0px, I'm not seeing much difference even if we set this.
Is 0px expected value?